base/api/testHelperServiceprovider.page.ts

```diff
@@ -3,7 +3,11 @@ import { Page, expect } from '@playwright/test';
 const FRONTEND_URL: string | undefined = process.env.FRONTEND_URL || "";
 
 export async function getSPId(page: Page, nameSP: string): Promise<string> {
-    const response = await page.request.get(FRONTEND_URL + `api/provider/all`, {});   
+    let response = await page.request.get(FRONTEND_URL + `api/provider/all`, {});   
+    if(response.status() != 200) {
+        console.log(response);
+        response = await page.request.get(FRONTEND_URL + `api/provider/all`, {}); 
```

Comment:
Warum wird der Request 2 Mal ausgeführt wenn der erste fehlgeschlagen? :)

Reply:
dieses hat eher temporären Charakter; wir haben an vielen Stellen mit dem BE Probleme und hier fange ich mal mit der Analyse an.